sentry_sdk/integrations/aws_lambda.py

```diff
@@ -214,3 +224,32 @@ def _get_url(event, context):
     if proto and host and path:
         return "{}://{}{}".format(proto, host, path)
     return "awslambda:///{}".format(context.function_name)
+
+
+def _get_cloudwatch_logs_url(context, start_time):
+    # type: (LambdaContext, datetime) -> str
+    """
+    Generates a CloudWatchLogs console URL based on the LambdaContext object
+
+    Arguments:
+        context {LambdaContext} -- context from lambda handler
+
+    Returns:
+        str -- AWS Console URL to logs.
+    """
+    start_time_str = (start_time - timedelta(seconds=1)).isoformat().split(".")[0]
```

Comment:
would prefer a strftime with explicit args here

Reply:
Ok, why? Happy to do it, just curious. could also do this:

`.replace(microsecond=0).isoformat()` if you object to the `.split('.')[0]`

Reply:
just personal preference